strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/configuration/ConfigurationManager.java

```diff
@@ -136,13 +131,12 @@ public String getRepositoryId(String storageAndRepositoryId)
     @Override
     public Configuration getConfiguration()
     {
-        return (Configuration) super.getConfiguration();
+        return configurationRepository.getConfiguration();
     }
 
-    public Resource getConfigurationResource()
-            throws IOException
-    {
-        return configurationResourceResolver.getConfigurationResource("repository.config.xml", "etc/conf/strongbox.xml");
+    @Override
+    public Resource getConfigurationResource() throws IOException {
+        return null;
```

Comment:
@ivanursul,

I think this should be as it was...
The idea is to be able to import a `strongbox.xml` if one is on the path, or specified as via the parameter. If there is no such specified (and there is no configuration yet in OrientDB), then load the one from the classpath (available in the `strongbox-storage-api-resources` artifact).